LICENSE-MIT

```diff
@@ -1,4 +1,4 @@
-Copyright 2018 Developers of the Rand project
+Copyright 2018-2024 Developers of the Rand project
 Copyright (c) 2014 The Rust Project Developers
```

Comment:
Can we change this from "Rand project" to "the rust-random project developers"? That at least is searchable.

Reply:
Done.